ethereumj-core/build.gradle

```diff
@@ -139,7 +139,7 @@ dependencies {
     compile "com.h2database:h2:1.4.187"
     compile "com.typesafe:config:1.2.1"
 
-    compile "org.mapdb:mapdb:2.0-beta12"
+    compile "org.mapdb:mapdb:2.0-beta13"
 
     compile "org.slf4j:slf4j-log4j12:${slf4jVersion}"
     compile "log4j:apache-log4j-extras:${log4jVersion}"
```

Comment:
Does this apply for all dependencies? Usually e.g. Spring provides security related fixes with some of the minor updates. Also other libraries might provide security related fixes that are not even announced by their vendors. I would feel uncomfortable to run a server that uses outdated libraries. Deserialization issues can compromise the underlying machine.